src/wasm32_bindgen.rs

```diff
@@ -13,15 +13,22 @@ use core::cell::RefCell;
 use core::mem;
 use std::thread_local;
 
+use js_sys::Uint8Array;
+// We have to rename wasm_bindgen to bindgen in the Cargo.toml for backwards
+// compatibility. We have to rename it back here or else the macros will break.
+extern crate bindgen as wasm_bindgen;
 use wasm_bindgen::prelude::*;
 
 use crate::error::{BINDGEN_CRYPTO_UNDEF, BINDGEN_GRV_UNDEF};
 use crate::Error;
 
+// Maximum is 65536 bytes see https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues
+const BROWSER_CRYPTO_BUFFER_SIZE: usize = 256;
+
```

Comment:
It's worth to rename this constant to a more appropriate name and remove the deprecated comment about 64 KiB.